website/docs/r/access_rule.html.markdown

```diff
@@ -10,6 +10,16 @@ description: |-
 
 This resource allows you to create and manage access rules, to give users and/or service accounts permissions on directories.
 
+## Example Usage
+
+```terraform
+resource "secrethub_access_rule" "example_app" {
+  account_name = "${secrethub_service_aws.demo_app.id}"
```

Comment:
Do we want to create the service account in the example as well, or do we want to limit the example to just this resource?

Reply:
I'd say let's just limit the example to this resource.